samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
Well I would, but there'd be too many breaking changes. The author of the [original grammar](https://github.com/StoneCypher/sublime-apl) used subpattern names that don't comply with [TextMate's recommendations on naming conventions](http://manual.macromates.com/en/language_grammars#naming_conventions)... e.g., if I were to somehow integrate my work into his, there'd be just as much (if not more) work involved with migrating the choice of class names to ensure there're no unexpected changes to users of the Sublime package.

[This is the grammar I worked on](https://github.com/Alhadis/language-apl). Aside from ironing out edge cases in numeral handling, it also covers a much wider range of keywords and has [partial support for heredocs](https://github.com/Alhadis/language-apl/blob/master/tests/gnu-web.apl#L127) (I only say "partial" because I wasn't able to set dynamic embedded highlighting based on a secondary parameter... I literally pushed TextMate's engine to its limit).

The license is ISC, functionally identical to the MIT license. I proudly release all my work under the most permissive license as possible, because restrictions are for pussies.